### PR TITLE
Support axo Releases

### PR DESCRIPTION
**Pull request by manyinsects**
Adds support for axo Releases (currently in closed beta) as a release data source for oranda. You can toggle between GitHub or axo Releases like this:

```json
{
  "components": {
    "source": "axodotdev"
  }
}
```

oranda will still default to GitHub, so no changes needed if you don't use axo Releases.

[gazenot]: https://github.com/axodotdev/gazenot

### CODE REVIEW

**Comment by manyinsects** [comment]
(for easier reviewing, I broke the commit that homogenizes line endings out of the other work, the complete diff view is unreadable, but if you go commit by commit, it should be OK)

File: oranda-css/css/pages/changelog.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
 }
 
 .release-body {
+  margin-top: 2rem;
```
Comment (Gankra) [inline oranda-css/css/pages/changelog.css:6]: ?

**Review by Gankra — APPROVED** [review]
looks right

File: oranda-css/css/pages/changelog.css (hunk 1 above)
Comment (manyinsects) [reply oranda-css/css/pages/changelog.css:6]: drive-by fix for the changelog title and body being too close

**Pull request merged**